c++/src/kj/async-inl.h

```diff
@@ -1476,6 +1476,10 @@ inline PromiseForResult<Func, void> evalLater(Func&& func) {
   return _::yield().then(kj::fwd<Func>(func));
 }
 
+inline Promise<void> evalLaterSimple() {
```

Comment:
I think `evalLater` no longer makes sense as a name if we're not passing anything to actually evaluate, and the meaning of "simple" isn't very clear either.

How about we actually name this `yield()`? So we're just making the existing function public?

We should also give `evalLast()` the same treatment I think. Its private implementation is currently called `yieldHarder()`, but that's obviously not a good public name. Maybe `yieldUntilQueueEmpty()`.